src/pages/Landing/components/StudentTicket/History/History.module.scss

```diff
@@ -0,0 +1,127 @@
+.History{
+   
+    @mixin text {
+     
+font-family: "Muli Bold";
+                font-size: 16px;
+                font-weight: bold;
+                line-height: 1.25;
+                letter-spacing: normal;
+                text-align: left;
+                color: #000000;       
+    }
+    .History_Text{
```

Comment:
Plz format scss, js files by extensions and fix all classNames to upper camel case